pset-functions.js

```diff
@@ -174,6 +271,32 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
+/* alright (your name here). Here's what's happening.
+// string is being passed through a function with a parameter of paladinword
+// the function is splitting the word into an array
+// there is a function of p checker that is checking the if the string and the value is strict equals.
+// if it is then return the value of true, if not return false
+// THATS IT!!! WOHOO >(^.^)< */
+
+
+function isPalindrome(paladinWord) {
```

Comment:
🛠 Read the comment below and update your code accordingly

@Defy1x fix the indentation issues, also please adhere to the best practice approach of structuring `if..else` statements

```
if (condidtion) { 

} else { 

}
```

Also, having an inner function `paladinChecker()` works, but isn't necessary and adds additional complexity that isn't needed. The `if..else` inside `paladinChecker()` could be used inside `isPalindrome` without the use of an inner function. Try implementing this witouth the 'palidinChecker()` function

Reply:
Done and done